userinterface/planui.ui

```diff
@@ -118,6 +118,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QPushButton" name="Merge">
+        <property name="text">
+         <string>Merge with...</string>
```

Comment:
I would suggest to group the functions modifying the current route, e.g.:
![image](https://github.com/RISE-Dependable-Transport-Systems/WayWise/assets/2404625/0b25f412-4057-410f-9941-1bb2b6f2b01d)